content/configuration/source.md

```diff
@@ -13,9 +13,68 @@ The source settings contains:
 
 Sources are bound to specific channels in the communications configuration. To learn more, read the [Source and Executor Bindings](/configuration/communication/#source-and-executor-bindings) section.
 
+## General merging strategy
+
+A source essentially notifies a channel about events for configured resources filtered by specified namespaces.
+
+When a channel binds to more than one source, the notifications are merged across all sources.
```

Comment:
```suggestion
When a channel binds to more than one source, the resource notifications are merged across all sources.
```

it would be good to distinguish `resources` from `recommendations`